cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizer.java

```diff
@@ -278,4 +331,66 @@ private String getRequestingUser() throws IllegalArgumentException {
     LOG.trace("Got requesting principal {}", principal);
     return principal.getName();
   }
+
+  @VisibleForTesting
+  public Map<AuthorizationPrivilege, Boolean> cacheAsMap() {
```

Comment:
But can we keep this AuthorizationPrivilege class in cdap.security spi or whatever we expose for commons/util in this way we will not have code duplication. extensions which want to use can use it doesn't seem like an internal thing to not expose it